From 614dc46fcd1de2f6a3fa3c8a8014cb32dbd0d0fb Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@redhat.com>
Date: Tue, 18 Nov 2008 08:19:45 +1000
Subject: [PATCH 03/42] xfree86: without CONFIG_HAL, Auto{Add|Enable}Devices and AEI is false.

There's little chance that we'll get the input devices at runtime without HAL,
we might as well force the server to add mouse/kbd devices automatically -
just like in the olden days.

Signed-off-by: Peter Hutterer <peter.hutterer@redhat.com>
(cherry picked from commit ace38fafb062372dcd3d56378b5b8f86525c6241)
---
 hw/xfree86/common/xf86Globals.c |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/hw/xfree86/common/xf86Globals.c b/hw/xfree86/common/xf86Globals.c
index a5edd82..2003b33 100644
--- a/hw/xfree86/common/xf86Globals.c
+++ b/hw/xfree86/common/xf86Globals.c
@@ -134,9 +134,15 @@ xf86InfoRec xf86Info = {
     .kbdCustomKeycodes          = FALSE,
     .disableRandR               = FALSE,
     .randRFrom                  = X_DEFAULT,
+#ifdef CONFIG_HAL
     .allowEmptyInput            = TRUE,
     .autoAddDevices             = TRUE,
     .autoEnableDevices          = TRUE
+#else
+    .allowEmptyInput            = FALSE,
+    .autoAddDevices             = FALSE,
+    .autoEnableDevices          = FALSE
+#endif
 };
 const char *xf86ConfigFile = NULL;
 const char *xf86InputDeviceList = NULL;
-- 
1.6.0.4

